ansible/plugins/library/ssh_config.py

```diff
@@ -47,7 +47,11 @@
   HostName {{ metadata.ansible_host }}
   IdentityFile {{ metadata.ansible_ssh_private_key_file }}
   User {{ metadata.ansible_user or 'root' }}
-{{ '  ' + ssh_arg|multi_replace(args) + '\n' if ssh_arg }}
+{{
```

Comment:
I added this code to enable rendering the provided port to the config.
@gdams feel free to edit or remove if you don't like this.